salt/__init__.py

```diff
@@ -25,24 +29,65 @@ def find_module(self, module_name, package_path=None):
                 return self
         return None
 
-    def load_module(self, name):
+    def create_module(self, spec):
         if USE_VENDORED_TORNADO:
-            mod = importlib.import_module("salt.ext.{}".format(name))
+            mod = importlib.import_module("salt.ext.{}".format(spec.name))
         else:
             # Remove 'salt.ext.' from the module
-            mod = importlib.import_module(name[9:])
-        sys.modules[name] = mod
+            mod = importlib.import_module(spec.name[9:])
+        sys.modules[spec.name] = mod
         return mod
 
+    def exec_module(self, module):
+        log.error("exec_module %r", module)
```

Comment:
```suggestion
    def exec_module(self, module):
        log.debug("exec_module %r", module)
```
Why `error`?

Reply:
This should probably be removed all together.

Reply:
```Ok, but what exactly triggers the no-name-in-module```
@s0undt3ch We already had this conversation, about false positives with pylint 2.4.4 and py3.8, hence the pragmatic disable.  If we really want to get to the bottom of it, explain why the workflow is doing a pylint check on a file that is unchanged in this PR, sha256sum verified same as branch 3006.x

Reply:
done